DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -817,6 +824,36 @@ public void GetEligibleCoursesToAddToGroup_does_not_return_courses_already_in_gr
             result.Should().NotContain(c => c.CustomisationId == 2);
         }
 
+        [Test]
+        public void GetCategoriesForCentreAndCentrallyManagedCourses_returns_expected_categories()
+        {
+            // Given
+            const string categoryName = "Category";
+            A.CallTo(() => courseCategoriesDataService.GetCategoriesForCentreAndCentrallyManagedCourses(A<int>._))

```

Comment:
Should this use the specific centre Id for this test file rather than A<int>._ ? If so, this applies to the test below as well